typings/index.d.ts

```diff
@@ -3028,6 +3028,11 @@ declare module 'discord.js' {
     around?: Snowflake;
   }
 
+  type ChannelMention = `<#${Snowflake}>`;
+  type UserMention = `<@${Snowflake}>`;
+  type MemberMention = UserMention | `<@!${Snowflake}>`;
+  type RoleMention = '@everyone' | `<@&${Snowflake}>`;
```

Comment:
```suggestion
  type RoleMention = '@everyone' | '@here' | `<@&${Snowflake}>`;
```

Reply:
`@here` is not a role, while `@everyone` is the role that everyone has.
![image](https://user-images.githubusercontent.com/73136330/124263625-cc293600-db33-11eb-916a-f8ded6951e48.png)

Reply:
Ah yeah, true.

Reply:
the `@everyone` role is actually unrelated to the `@everyone` ping; actually mentioning the `@everyone` role outputs `@@everyone` and doesn't ping anyone.

would probably be best to remove that if/return tbh

Reply:
Well maybe, but I didn't change anything on that method. This is just to add types and `Role.toString()` can return `@everyone` too so I added it...